packages/astro/src/types/public/config.ts

```diff
@@ -2433,31 +2433,31 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 		 * @name experimental.liveContentCollections
 		 * @type {boolean}
 		 * @default `false`
-		 * @version 5.x
+		 * @version 5.10
 		 * @description
 		 * Enables the use of live content collections.
 		 *
 		 */
 		liveContentCollections?: boolean;
 
 		/**
-		 * @name experimental.rawEnvValues
+		 * @name experimental.staticImportMetaEnv
 		 * @type {boolean}
 		 * @default `false`
-		 * @version 5.12
+		 * @version 5.13
 		 * @description
 		 *
-		 * Disables coercion of `import.meta.env` values that are populated from `process.env`, allowing you to use the raw value.
+		 * Disables replacement of `import.meta.env` values with `process.env` calls and their coercion
```

Comment:
Should this sentence be updated too?

Reply:
I think that one is OK? I left it intentionally, figuring below helped with the extra details.